pyndustri.pyi

```diff
@@ -613,3 +613,6 @@ def sleep(secs: float):
     """
     Sleep for the given amount of seconds.
     """
+
+def inline(func: Callable[..., Any]) -> Callable[..., Any]:
```

Comment:
Wonder if we should use a `TypeVar` so that editors know that the returned function matches the input exactly. Or perhaps this works as it is? Haven't use `Callable` that much.